app/views/accommodations/show.html.erb

```diff
@@ -1,10 +1,6 @@
-<div class="row">
-  <h3>Editing accommodation '<%= @accommodation.label %>' in building '<%= @accommodation.building.label %>'</h3>
-</div>
+<h3>Editing accommodation '<%= @accommodation.label %>'</h3>
 
-  <div class="row">
-    <%= render 'layouts/error_messages' %>
-  </div>
+<%= render 'layouts/error_messages' %>
```

Comment:
these extra row divs cause these elements to be -15 pixels to the left -- in general, row classes should be used with column classes, or neither should be used.  aka. it should be a 1:1 ratio.